pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -221,9 +221,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	isUpdate := app.Status.Workflow.Message != "" && workflowInstance.Status.Message == ""
 	workflowInstance.Status.Phase = workflowState
 	app.Status.Workflow = workflow.ConvertWorkflowStatus(workflowInstance.Status, app.Status.Workflow.AppRevision)
+	logCtx.Info("Workflow return state=%s", workflowState)
```

Comment:
remove this debug log

Reply:
This is not a debug log. It is a refactor of previous logs.